about.html

```diff
@@ -0,0 +1,150 @@
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./styles/styles.css">
+</head>
+
+<body>
+    <nav class="sub-nav bg-secondary">
+        <div class="container">
+            <ul class="sub-nav-list float-right">
+                <li class="sub-nav-item"><i class="fa-brands fa-facebook-f"></i></li>
+                <li class="sub-nav-item"><i class="fa-brands fa-twitter"></i></li>
+                <li class="sub-nav-item">English</li>
+                <li class="sub-nav-item">My Page</li>
+                <li class="sub-nav-item">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <header class="navbar fixed-top-nav" id="top-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-icon"></i>
+            <div class="desktop-nav">
+                <a href="index.html"><img class="logo-nav" src="./images/aspire-logo.png" width="110px"
+                        alt="Aspire logo"></a>
+                <ul class="nav-list float-right">
+                    <li><a class="nav-list-item" href="index.html">Home</a></li><br>
+                    <li><a class="nav-list-item" href="#about-myself">Program</a></li><br>
+                    <li><a class="nav-list-item" href="#my-form">Join</a></li><br>
+                    <li><a class="nav-list-item" href="#my-form">Sponsor</a></li><br>
+                    <li><a class="nav-list-item" href="#my-form">News</a></li><br>
+                    <li><a href="#" class="mt-3  p-2" id="btn-nav">Company</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </header>
+
+
+    <main class="main-section text-center">
+        <section class="top-bg">
+            <section class="hero container">
+                <br>
+                <h4 class="greetings">"Explore Your Dream Career with Aspire"</h4>
+                <h2 class="hero-title">Get connected <br /> to the experts Globally</h2>
+                <br>
+            </section>
+
+            <section class="message">
+
+                <div class="container">
+                    <p class="message-p p-3 border border-white">
+                        At Career Guiders, we help young people explore different career options with the help of
+                        industry experts. Our mission is to reduce unemployment in Africa by funding talents and
+                        connecting young people with experts in their field of interest. Join us today and start your
+                        journey to success!
+                    </p>
+                    <p>please contact us per Email for any further questions.</p>
+                    <b class="underline">komezarsene15@gmail.com</b>
+                </div>
+            </section>
+            <br><br><br>
+        </section>
```

Comment:
Your design
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/bf9e6342-ae17-4749-ad3e-5de715d44430)

Guideline
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/321ce204-1902-4720-9f54-0dd13ba14749)

According to the requirements
- We should follow the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements.

- Please kindly modify your design to look almost identical to the original design
   - Kindly ensure that the pages use the exact same colors, for text, backgrounds, borders, spacing etc, from the original design.
   - Basic color swatch: #272A31, #EC5242, #D3D3D3

Font face
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/617533bc-754f-45d7-b47f-dd4b62efe6c8)